.travis.yml

```
@@ -5,6 +5,18 @@ language: go
 
 install: true
 
+branches:
+  only:
+    - master
+
+before_install:
+  - |
+      if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(.md)|(.html)|^(LICENSE)|^(docs)'
+      then
+        echo "Only docs were updated, not running the CI."
+        exit
```

```suggestion
        exit 0
```
Minor. You can use `0` to be explicit that it's all good.